modules/buttons/info.html

```diff
@@ -0,0 +1,33 @@
+<style>
+    .info {background-color: #2196F3;} /* Blue */
+    /*.info:hover {background: #0b7dda;}*/
+    button{
+        color: white; 
+        border-radius: 4px;
+        text-align: center;
+        border: none;
+        padding: 14px 28px;
+    }
+    .icon,.text {
+        vertical-align: middle;
+        display: inline-block;
+        align-items: center;
+    }
+    
+</style>
+
+<body>
+    <div>
+        <button class="info">
+         <span class="icon">
+            <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+                width="20" height="20"
+                viewBox="0,0,256,256"
+                style="fill:#000000;">
+                <g fill="#ffffff" fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none" font-size="none" text-anchor="none" style="mix-blend-mode: normal"><g transform="scale(5.33333,5.33333)"><path d="M24,4c-11.02793,0 -20,8.97207 -20,20c0,11.02793 8.97207,20 20,20c11.02793,0 20,-8.97207 20,-20c0,-11.02793 -8.97207,-20 -20,-20zM24,7c9.40662,0 17,7.59339 17,17c0,9.40661 -7.59338,17 -17,17c-9.40661,0 -17,-7.59339 -17,-17c0,-9.40661 7.59339,-17 17,-17zM24,14c-1.10457,0 -2,0.89543 -2,2c0,1.10457 0.89543,2 2,2c1.10457,0 2,-0.89543 2,-2c0,-1.10457 -0.89543,-2 -2,-2zM23.97656,20.97852c-0.82766,0.01293 -1.48843,0.69381 -1.47656,1.52148v11c-0.00765,0.54095 0.27656,1.04412 0.74381,1.31683c0.46725,0.27271 1.04514,0.27271 1.51238,0c0.46725,-0.27271 0.75146,-0.77588 0.74381,-1.31683v-11c0.00582,-0.40562 -0.15288,-0.7963 -0.43991,-1.08296c-0.28703,-0.28666 -0.67792,-0.44486 -1.08353,-0.43852z"></path></g></g>
+            </svg>
+         </span>
+         <span class="text">Info</span>
+        </button>
+    </div>
+</body>
```

Comment:
Finally, To increase the level of acceptance of your PR, Always help me with a screenshot so I don't have to create a branch or check out the entire stuff here. LOL. Nice one.

Reply:
sure.